packages/ui/src/styles/search.css

```
@@ -0,0 +1,45 @@
+  justify-content: center;
+  width: 44px;
+  height: 44px;
+  flex-shrink: 0;
+  border: 0;
+  border-radius: 999px;
+  background: transparent;
+  color: inherit;
+  cursor: pointer;
+}
+.prisma-search-trigger:hover {
+  background: color-mix(in srgb, currentColor 8%, transparent);
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)